code/modules/mob/living/silicon/robot/robot_vr.dm

```diff
@@ -17,7 +18,7 @@
 
 /mob/living/silicon/robot/updateicon()
 	..()
-	if(icon == 'icons/mob/widerobot_vr.dmi' || 'icons/mob/64x64robot_vr.dmi')
+	if(dogborg == 1)
 		if(stat == CONSCIOUS)
 			if(sleeper_g == 1)
```

Comment:
No need to nest these.
`if(dogborg == TRUE && stat == conscious && sleeper_g == TRUE)`